eye2bids/edf2bids.py

```diff
@@ -520,8 +525,11 @@ def edf2bids(
 
     events_json.input_file = input_file
     events_json.two_eyes = _2eyesmode(df_ms_reduced)
+    events_json.TaskName = _has_TaskName(events)
+
+    if events_json.TaskName:
+        events_json["TaskName"] = _extract_TaskName(events)
```

Comment:
```suggestion
    if TaskName := _extract_TaskName(events)
        events_json["TaskName"] = TaskName
```

Reply:
walrus operator `:=` in this case is equivalent to 

```python
TaskName = _extract_TaskName(events)
if TaskName:
   ...
```

and the empty string `""` will evaluate as a falsy.

```python
>>> if not  "":
...     print('foo')
... 
foo
```